src/pages/Browse/index.tsx

```diff
@@ -138,6 +139,22 @@ export class BrowseBase extends React.Component<Props> {
       );
     }
 
+    const getContent = () => {
```

Comment:
Since this doesn't *need* to be a closure, I think it would be slightly nicer as an instance method.